pkg/prometheus/promcfg.go

```diff
@@ -683,7 +683,7 @@ func (cg *ConfigGenerator) generatePodMonitorConfig(
 		relabelings = append(relabelings, yaml.MapSlice{
 			{Key: "action", Value: "keep"},
 			{Key: "source_labels", Value: []string{"__meta_kubernetes_pod_label_" + sanitizeLabelName(k), "__meta_kubernetes_pod_labelpresent_" + sanitizeLabelName(k)}},
-			{Key: "regex", Value: fmt.Sprintf("%s;true", m.Spec.Selector.MatchLabels[k])},
+			{Key: "regex", Value: fmt.Sprintf("(%s);true", m.Spec.Selector.MatchLabels[k])},
```

Comment:
I wonder why these changes are still shown in the diff. They should already be in the main branch :thinking:

Reply:
Ok, looks like they were never merged into main.

Reply:
[They are not](https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/prometheus/promcfg.go#L684-L686). The PR introducing this (https://github.com/prometheus-operator/prometheus-operator/pull/4395) was merged to `release-0.52` and from what I see the idea was that we will bring it into `main` during syncing 0.52.1 into `main` (so this PR).